lib/index.js

```diff
@@ -11,6 +11,7 @@ var crypto = require('crypto');
 var database = require('bedrock-mongodb');
 var util = require('util');
 var BedrockError = bedrock.util.BedrockError;
+require('bedrock-server');
```

Comment:
Yes.
- Add key module config var for the base uri.
- That var can be set explicitly if needed.  For instance if module is used w/o bedrock-server but with known value.
- Default will be a computed config value:
  - Default to bedrock-server baseUri config if it exists.
  - Else default to some generic thing like `urn:key:`.

I see two issues with a fallback to `urn:key:`:
- Keys are not generated with globally unique ids so these ids could be problematic That could be solved with a UUID or similar mode for ID generation vs distributed IDs used now.
- The createPublicKeyId(name) call would create ids like `urn:key:/keys/{name}`, or if key.basePath was set as empty, then `urn:key:/{name}`.
Both of these issues could be put off for now since they are not an issue with current projects.  May need to add a way to be more flexible in how ids are generated.  This issue could come up with other modules that also have server variable based id generation.